tests/all_sh_components.txt

```diff
@@ -206,3 +206,19 @@ component_test_cmake_shared () {
     cd "$TF_PSA_CRYPTO_ROOT_DIR"
     rm -rf "$OUT_OF_SOURCE_DIR"
 }
+
+component_check_recursion () {
+    msg "Check: recursion.pl" # < 1s
+    tests/scripts/recursion.pl library/*.c
```

Comment:
library/*.c is not suitable

Reply:
Sorry, I meant to change that to core and I clearly forgot.